Find a functionally equivalent code:android.os.Bundle.getShort:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or defaultValue if
 * no mapping of the desired type exists for the given key.
 *
 * @param key a String
 * @param defaultValue Value to return if key does not exist
 * @return a short value
 */

@@ -1,13 +1,3 @@
 {
-    unparcel();
-    Object o = mMap.get(key);
-    if (o == null) {
-        return defaultValue;
-    }
-    try {
-        return (Short) o;
-    } catch (ClassCastException e) {
-        typeWarning(key, o, "Short", defaultValue, e);
-        return defaultValue;
-    }
+    return super.getShort(key, defaultValue);
 }

